extensions/amp-base-carousel/1.0/scroller.js

```diff
@@ -100,26 +111,26 @@ function ScrollerWithRef(
     ignoreProgrammaticScrollRef.current = true;
     setStyle(container, 'scrollBehavior', 'auto');
     let position;
+    const offset = container./* OK */ offsetWidth / visibleCount;
```

Comment:
Elsewhere you call `offsetWidth/visibleCount` a `slideWidth`. Should we do it here as well?

Reply:
Renamed here as well as in `advance`